Python/bigtable/read_bigtable.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import apache_beam as beam
+
+from apache_beam import Map
+from apache_beam.io.gcp.bigtableio import ReadFromBigtable
+from apache_beam.options.pipeline_options import PipelineOptions
+
+
+def run():
+    """
+    This Apache Beam pipeline retrieves data from Google Cloud Bigtable table.
+    It performs the following steps:
+    1. Read from Bigtable:
+        - The pipeline reads data from a specified Bigtable instance and table.
+    2. Data Transformation:
+        - The rows retrieved from Bigtable are transformed to extract cell data.
+    3. Logging Data:
+        - The extracted cell data is logged.
+    """
+    class BigtableOptions(PipelineOptions):
+        @classmethod
+        def _add_argparse_args(cls, parser):
+            parser.add_argument(
+                '--project_id',
+                default="apache-beam-testing",
+                help='Input project ID'
```

Comment:
I see that in other example help text just expains what is expected, so I suggest remove the word "Input" from the help string.